test/simulator/test/org/apache/cassandra/simulator/test/AccordJournalSimulationTest.java

```diff
@@ -165,35 +166,35 @@ public static class State
             }
         }
         private static final ExecutorPlus executor = ExecutorFactory.Global.executorFactory().pooled("name", 10);
-        private static final AccordJournal journal = new AccordJournal();
+        private static final AccordJournal journal = new AccordJournal(null);
         private static final int events = 100;
         private static final CountDownLatch eventsWritten = CountDownLatch.newCountDownLatch(events);
         private static final CountDownLatch eventsDurable = CountDownLatch.newCountDownLatch(events);
         private static final List<Throwable> exceptions = new CopyOnWriteArrayList<>();
 
         static
         {
-            journal.start();
+            journal.start(null);
         }
 
         public static void append(int event)
         {
             TxnRequest<?> request = toRequest(event);
-            journal.appendMessage(request, executor, new AsyncWriteCallback()
-            {
-                @Override
-                public void run()
-                {
-                    durable(event);
-                }
-
-                @Override
-                public void onFailure(Throwable error)
-                {
-                    eventsDurable.decrement(); // to make sure we don't block forever
-                    exceptions.add(error);
-                }
-            });
+//            journal.appendMessageTest(request, executor, new AsyncWriteCallback()
```

Comment:
Test not passing?

Reply:
I need to check if things work again, but there was a bug with how interceptors were set up in the initial simulation test implementation that caused things to hang, and the test had to be disabled. I will see what's needed to make it work.